t/unit/backends/test_base.py

```diff
@@ -198,6 +203,21 @@ def test_json_exception_arguments(self):
         y = self.b.exception_to_python(x)
         assert isinstance(y, Exception)
 
+    def test_json_exception_nested(self):
+        if sys.version_info < (3, 3):
+            raise pytest.skip('Not supported on this Python version')
+        self.b.serializer = 'json'
+        print(repr(objectexception.Nested('msg')))
+        x = self.b.prepare_exception(objectexception.Nested('msg'))
+        assert x == {
+            'exc_message': ('msg',),
+            'exc_type': 'objectexception.Nested',
+            'exc_module': objectexception.Nested.__module__}
+        print(x)
+        y = self.b.exception_to_python(x)
+        print(repr(y))
```

Comment:
are the print() functions intended here?

Reply:
no, I will remove them